content/ideas/algebraic-graphs.md

```diff
@@ -0,0 +1,61 @@
+---
+title: More graph algorithms for Alga
+---
+
+The [Algebraic Graphs library](https://github.com/snowleopard/alga) has so far succeeded
+(thanks in part to last years summer of code) in providing a common, 
+typesafe interface for graph instances. However, general ecosystem adoption remains low,
+in part due to the lack of graph algorithms provided.
```

Comment:
I would replace the last sentence by "However, a lot of graph algorithms are still missing".